### DISCUSSION TITLE
Welcome to meerk40t Discussions!

### DISCUSSION THREAD

**Discussion by tatarize** [meerk40t/meerk40t#1316] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andrea74471** [comment]
Non sono un esperto di pc quale versione scaricare x un pc portatile resetteto di recente?
